pom.xml

```diff
@@ -95,10 +95,15 @@
         https://dev.to/scottshipp/how-to-fix-a-mockito-cannot-mock-this-class-exception-in-a-spring-boot-app-457e -->
         <dependency>
             <groupId>org.mockito</groupId>
-            <artifactId>mockito-inline</artifactId>
-            <version>4.6.1</version>
+            <artifactId>mockito-core</artifactId>
+            <version>5.5.0</version>
```

Comment:
the indentation looks a little off. can you fix it please

Reply:
Done.